xarray/core/formatting.py

```diff
@@ -211,9 +211,41 @@ def format_array_flat(array, max_width):
     return pprint_str
 
 
-def summarize_variable(
-    name, var, col_width, show_values=True, marker=" ", max_width=None
-):
+def inline_dask_repr(array):
+    """Similar to dask.array.DataArray.__repr__, but without
+    redundant information that's already printed by the repr
+    function of the xarray wrapper.
+    """
+    assert isinstance(array, dask_array_type), array
+    chunksize = tuple(c[0] for c in array.chunks)
+    return "dask.array<shape={}, chunksize={}>".format(array.shape, chunksize)
```

Comment:
Should we remove shape here and in the sparse repr? https://github.com/pydata/xarray/issues/2528#issuecomment-434456201

Also what about adding support for `max_width` (and closing #2528)?

Reply:
Sure, we could drop `shape` if we think it's redundant in general, or potentially only drop it if the repr is already too long to fit into `max_width`.

I don't know a good way to truncate to `max_width` in general, beyond blinding truncating the repr to a certain maximum width and adding `...`.

Reply:
+1 for dropping shape